docs/source/pipelines.rst

```diff
@@ -24,30 +24,11 @@ ML engineers, and DevOps teams to iterate very quickly within their domains of e
 offers :ref:`APIs <pipelines-apis>` and a :ref:`CLI <cli>` for executing pipelines and inspecting
 their results.
 
-Installation
-------------
-MLflow Pipelines is available as an extension of the
-`MLflow Python library <https://pypi.org/project/mlflow/>`_. You can install MLflow Pipelines
-from PyPI as follows:
-
-.. code-block:: sh
-
-    pip install mlflow[pipelines]  # for pip
-    conda install -c conda-forge mlflow-pipelines  # for conda
-
-.. note::
-  MLflow Pipelines requires Make, which may not be preinstalled on some systems (e.g. Windows).
-  Please ensure Make is installed before using MLflow Pipelines.
-
-.. note::
-  You can install MLflow Pipelines from a Databricks Notebook by running
-  ``%pip install mlflow[pipelines]`` or install MLflow Pipelines on a Databricks Cluster
-  by following the instructions at
-  https://docs.databricks.com/libraries/cluster-libraries.html#install-a-library-on-a-cluster.
-
-  Note that `Databricks Runtime <https://docs.databricks.com/runtime/dbr.html>`_ version 11.0
-  or greater is required in order to install MLflow Pipelines on Databricks.
```

Comment:
Does this section need to be kept for Databrick Runtimes where mlflow>=2.0 is not pre-installed?

Reply:
```
.. note::
  You can install MLflow Pipelines from a Databricks Notebook by running
  ``%pip install -U mlflow`` or install MLflow Pipelines on a Databricks Cluster
  by following the instructions at
  https://docs.databricks.com/libraries/cluster-libraries.html#install-a-library-on-a-cluster.

  Note that `Databricks Runtime <https://docs.databricks.com/runtime/dbr.html>`_ version 11.0
  or greater is required in order to install MLflow Pipelines on Databricks.
```

Reply:
I think we should rephrase this to say that MLP requires DBR 11+, MLP is preinstalled on MLR 12+, and users can install MLP in DBR 11 via `pip install mlflow>=2.0` or `pip install mlflow[pipelines]<2.0` (if they want to remain on a 1.x version).

Reply:
Sounds good!